docs/handlers.md

```diff
@@ -54,13 +56,13 @@ Library exports a `State` object that provides a number of constants used to exp
 
 #### `NativeViewGestureHandler` extra properties
 
- - `shouldActivateOnStart`
+ - `shouldActivateOnStart` (**Android only**)
```

Comment:
If yes, I think it's rather a bug in the code, not in docs 😃

Reply:
I sort of agree - it's not technically a bug in the code. Rather, the property in the Android codebase is simply not linked up to anything. So it's more like it's unimplemented.